### DISCUSSION TITLE
.bib file with 'pagetotal ={}' entry

### DISCUSSION THREAD

**Discussion by nagys5** [jgm/pandoc#9423] [General]
Recently I discovered the bibliographic possibilities of pandoc and markdown. Until now, I wrote the references in LaTeX, so I have several files with the .bib extension, mostly with Hungarian literature.
I tried to reuse these using the CSL style. It worked, except for the items where the entry was something like this: `pagetotal = {123}`
I realized that this kind of field is used by CSL as `number-of-pages`, so I tried to do the pandoc's biblatex to csljson conversion:
`pandoc -f biblatex -t csljson -o bibliography.csljson bibliography.bib`
__The `pagetotal` or `number-of-pages` line was missing from the result!__
I believe this may be because the conversion is literal and not logical. If so, please fix this bug unless there is no other solution.
_Note:_ I didn't check for the same error among other keywords.

**Comment by jgm** [comment]
I just pushed a commit that fixes this, thanks for noting it!

**Reply by nagys5** [reply]
Thank you,
how can I use it? I don't know the Haskell language. The version of install files is the old one: 3.1.11.1

**Reply by jgm** [reply]
Wait for a new release, or you can try a nightly tomorrow, or compile from source (see installing instructions on website).
